### DISCUSSION TITLE
Option to disable the registration

### DISCUSSION THREAD

**Discussion by ridhof** [teableio/teable#428] [Ideas]
first of all, congrats on the teable open source release!
compared to any other similar projects, teable picks my interest the most. so I couldn't help it to try it right away.

after running in local, and do a quick trial and error, I found no issues yet.
but then I realized after I deployed teable publicly, there is no option to disable the registration. yes I would like to self-host teable, but I want the selfhosted teable I had is only used by people I know. I don't want a random people to signing up at my teable.

there are plenty of way to achieve this without changing the teable source code for sure, by using vpn or some sort. but I am wondering if this option will be available in the code base through a flag or environment variable.

would be happy to receive any feedback!

**Comment by tea-artist** [comment]
Thank you for the experience, Are there other open source project practices that can be referenced?

**Reply by ridhof** [reply]
hi, an open source project that follows such practice that I am currently using is https://github.com/sabre-io/Baikal

tl;dr 
when running the software or do the deployment for the first time, you are asked for the admin credential. but once the admin credential already setup, it only show a login option, no sign up or registration. see here https://sabre.io/baikal/install/

**Reply by tea-artist** [reply]
We may be able to solve this problem by providing OIDC(#434 ) Login so that you can control the authentication source

**Reply by mz0in** [reply]
Hi ,  OIDC or providing other sign up alternative is not only the case , an admin should have the option and tools on more control on the user registration process , the same as implemented in Api , but with more user access role driven 
I added  a comment on OIDC(https://github.com/teableio/teable/issues/434#issuecomment-1994138086 )
